incubator/vault/values.yaml

```diff
@@ -194,11 +196,15 @@ vault:
   #   secret:
   #     secretName: some-secret
   liveness:
+    # DR requires Vault enterprise
+    drEnabled: false
```

Comment:
I'm wondering if it might make sense to follow the naming convention similar to `readyIfStandby` and call this `readyIfDRSecondary` instead of `drEnabled`?

Reply:
Good suggestion. I'll make the changes.

Reply:
Hi @mathplusyou , any chance that you could make the changes so that we can get the PR merged?

Reply:
@jbialy  I'll get to it this week!